core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -82,7 +83,7 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
 
     TestUtils.waitUntilTrue(() => {
       resultFuture.isDone && !resultFuture.isCancelled && !resultFuture.isCompletedExceptionally
-    }, msg = s"Could not commit the offsets successfully. Last response ${resultFuture.get()}.")
+    }, msg = s"Could not commit the offsets successfully. Last response ${resultFuture.get(0, TimeUnit.MILLISECONDS)}.")
   }
```

Comment:
Sorry. I meant that you could replace this entire code by `resultFuture.get(JTestUtils.DEFAULT_MAX_WAIT_MS, TimeUnit.SECONDS)`.

Reply:
Another question, could we use `commitOffset` method?

Reply:
Yeah I didn't realize that. Just changed to use `commitOffset`